onnx/reference/ops/op_scan.py

```diff
@@ -96,17 +96,7 @@ def _common_run_shape(self, *args):  # type: ignore
             states,
         )
 
-    def _run(  # type:ignore
-        self,
-        *args,
-        body=None,  # noqa: ARG002
```

Comment:
I think we need to keep some of these for them to match the op definition. Are there better ways to handle them?

Reply:
It’s better to keep them than hiding under args and kwargs for readability

Reply:
Because these arguments are not used now, leaving them alone may cause confusion.